src/styles/blocks/card.scss

```diff
@@ -0,0 +1,81 @@
+.card {
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+  display: flex;
```

Comment:
It's not necessary to use `flex` and `flex-direction: column` for the `.card` class. By default, block elements are arranged in a column. You can remove these properties.